bcda/testUtils/client.go

```diff
@@ -64,7 +64,8 @@ func (bbc *BlueButtonClient) GetData(endpoint, patientID string) (string, error)
 			}
 			cleanData := strings.Replace(string(fData), "20000000000001", patientID, -1)
 			if bbc.MBI != nil {
-				cleanData = strings.Replace(cleanData, "c945694486322cfcf792a4d1c69ea88bf0c134edaf9f1d68a18466f9dc6ec2fe", client.HashIdentifier(*bbc.MBI), -1)
+				// no longer hashed, but this is only a test file with synthetic test data
+				cleanData = strings.Replace(cleanData, "-1Q03Z002871", *bbc.MBI, -1)
```

Comment:
Does this mean the unhashed MBI that is coming back is a negative value?  I'm not sure that is correct.  Looking at the list of MBIs loaded into the `prod-sbx`, I expect this to be non-negative.

Reply:
yeah i just put the negative on it to signify a purely synthetic value, i can take it off. But this code really only touches the synthetic patient static file found here: shared_files/synthetic_beneficiary_data/Patient_MBI

Reply:
Ah!  That makes sense.  Thanks for clarifying.  I think it is perfect as-is.  No need to change.